scripts/android/run-android.sh

```diff
@@ -2,4 +2,11 @@
 
 envprop() { grep "^${1}=" ./.env | cut -d'=' -f2 | head -n 1; }
 
-react-native run-android --appId "$(envprop 'ANDROID_APPLICATION_ID')"
\ No newline at end of file
+appId=$(envprop 'ANDROID_APPLICATION_ID')
+
+# Check if KETTLE_API_KEY is set in the .env file
+if [ "$(envprop 'KETTLE_API_KEY')" ]; then
+  react-native run-android --appId "$appId" --mode=beaconsDebug
```

Comment:
What does this part do? It sets `beaconsDebug` also in production for other users if that api key is available?

Reply:
Hi, this is when you run in your machine `yarn android` because now we have two variations app and beacons, by default it lunches app but since ATB has beacons enabled it fails because beacon implementation is not loaded, so to avoid this if API is set then switch the app to beacons, but only when developing.